vite.go

```diff
@@ -28,14 +29,15 @@ var (
 type Vite struct {
 	config *config.Config
 
-	walletManager *wallet.Manager
-	verifier      verifier.Verifier
-	chain         chain.Chain
-	producer      producer.Producer
-	net           net.Net
-	pool          pool.BlockPool
-	consensus     consensus.Consensus
-	onRoad        *onroad.Manager
+	walletManager      *wallet.Manager
+	verifier           verifier.Verifier
+	chain              chain.Chain
+	producer           producer.Producer
+	net                net.Net
+	pool               pool.BlockPool
+	consensus          consensus.Consensus
+	consensus_verifier interfaces.ConsensusVerifier
```

Comment:
Is `consensus_verifier` needed here? `consensus` implements `interfaces.ConsensusVerifier` and thus could be used instead.

Maybe also remove the `Verifier` interface in `ledger/consensus/consensus.go` and replace with `interfaces.ConsensusVerifier`.